settings.gradle.kts

```diff
@@ -139,8 +139,12 @@ include(":smoke-tests:images:servlet:servlet-5.0")
 include(":smoke-tests:images:spring-boot")
 
 include(":smoke-tests-otel-starter:spring-smoke-testing")
-include(":smoke-tests-otel-starter:spring-boot-3")
-include(":smoke-tests-otel-starter:spring-boot-3-reactive")
+include(":smoke-tests-otel-starter:spring-boot-non-reactive-common")
```

Comment:
If list the modules in lexicographic order, should list `:smoke-tests-otel-starter:spring-boot-non-reactive-common` behind `:smoke-tests-otel-starter:spring-boot-non-reactive-3`, same problem in line 145.